dapp/src/validateInputs.js

```diff
@@ -8,7 +8,8 @@ const schema = Joi.object({
   mailJetApiKeyPrivate: Joi.string().required(),
   mailJetSender: Joi.string().email().required(),
   emailSubject: Joi.string().required(),
-  emailContent: Joi.string().required(),
+  emailContentOrMultiAddr: Joi.string().required(),
```

Comment:
in the case where `emailContentEncryptionKey` exists, we may want to check that `emailContentOrMultiAddr` is a multiAddr.
```suggestion
  emailContentOrMultiAddr: Joi.alternatives().conditional(
    'emailContentEncryptionKey',
    {
      is: Joi.string().required(),
      then: Joi.string()
        .pattern(/^\/(ipfs|p2p)\//)
        .message(
          '"emailContentOrMultiAddr" must be a multiAddr when "emailContentEncryptionKey" is provided'
        )
        .required(),
      otherwise: Joi.string().required(),
    }
  ),
```